src/components/TextField/TextField.tsx

```diff
@@ -37,20 +46,25 @@ export const TextField: React.FC<Props> = ({
 
       <div className="control">
         <input
+          name={name}
           type="text"
           id={id}
           data-cy={`movie-${name}`}
           className={classNames('input', {
-            'is-danger': hasError,
+            'is-danger': hasError || errorLink || errorSpace,
           })}
           placeholder={placeholder}
           value={value}
-          onChange={event => onChange(event.target.value)}
+          onChange={onChange}
           onBlur={() => setTouched(true)}
```

Comment:
`onBlur={() => setTouched(true)} - It's not it?` - no 😄 

`const handleBlur = () => setTouched(true)`;

```suggestion
          onBlur={handleBlur}
```